tests/tests/scissor_tests/mod.rs

```diff
@@ -38,16 +38,18 @@ fn scissor_test_impl(ctx: &TestingContext, scissor_rect: Rect, expected_data: [u
             label: Some("Pipeline"),
             layout: None,
             vertex: wgpu::VertexState {
-                entry_point: "vs_main",
                 module: &shader,
+                entry_point: "vs_main",
```

Comment:
nitpick: It'd be nice to not move the  `entry_point` field here; that only seems like diff noise. This seems to happen in a few other tests as well.

Reply:
I thought this is small enough to be fine. These fields are always in that order in the other 80-90% of cases and I fixed the remaining ones.

Reply:
If this change is to be consistent with how things normally get formatted, then 👍🏻👍🏻. Minor complaint leading to new knowledge, yay!